etc/inc/notices.inc

```diff
@@ -350,6 +350,10 @@ function send_smtp_message($message, $subject = "(no subject)", $force = false)
 
 	if ($config['notifications']['smtp']['fromaddress']) {
 		$from = $config['notifications']['smtp']['fromaddress'];
+		preg_match('/[\w\s]+/', $from, $matches);
+		$from_display = trim($matches[0]);
+		if(preg_match('/(?:<)(.+)(?:>)$/', $from, $matches))
+			$from = $matches[1];
```

Comment:
What I meant is, without this change, you already are able to add both forms, there is no check preventing you to do it. And with the change, if you add a single email address (e.g. renato@pfsense.org) it's going to be modified to "renato <renato@pfsense.org>" because preg_match at line 353 is going to match every char before '@'.